src/test/erasure-code/CMakeLists.txt

```diff
@@ -83,9 +83,12 @@ target_link_libraries(unittest_erasure_code_plugin_jerasure
   global
   osd
   ec_jerasure
-  common)
+  common
+  ${UNITTEST_LIBS}
+  )
 add_dependencies(unittest_erasure_code_plugin_jerasure
   ec_jerasure)
+set_target_properties(unittest_erasure_code_plugin_jerasure PROPERTIES COMPILE_FLAGS ${UNITTEST_CXX_FLAGS})
```

Comment:
@tchaikov 
Oke, I see what you mean. It was already wierd, since this test is the only one missing the googletest addititons....  And this was in the jenkins/ceph-master build, I did not get the same error later in my WIP.
Running a build right now, but I'll get back on this.